### PR TITLE
add captcha connector

### PR DESCRIPTION
**Pull request by kspearrin**
Adds support for captcha connector that can be consumed via iframe from other Bitwarden clients for solving HCaptcha challenges.

### CODE REVIEW

File: src/connectors/captcha.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8" />
+    <title>Bitwarden Captcha Connector</title>
+    <script src="https://hcaptcha.com/1/api.js"></script>
+</head>
+
+<body style="background: transparent;">
```
Comment (addisonbeck) [inline src/connectors/captcha.html:10]:
```suggestion
<body>
```

File: src/connectors/captcha.scss [hunk 2]
```diff
@@ -0,0 +1,5 @@
+body {
+    min-width: 0px !important;
+    padding: 0;
+    margin: 0;
+}
```
Comment (addisonbeck) [inline src/connectors/captcha.scss:5]:
```suggestion
body {
    min-width: 0px !important;
    padding: 0;
    margin: 0;
    background: transparent;
}
```

**Review by addisonbeck — COMMENTED** [review]
Minor nit, just seems like you don't need to do an inline style here.

**Comment by kspearrin** [comment]
> 
> 
> Minor nit, just seems like you don't need to do an inline style here.

Accepted your suggestions.

**Review by addisonbeck — APPROVED** [review]

**Pull request merged**